lib/defaults/defaults.go

```diff
@@ -355,7 +355,7 @@ const (
 	SerfBin = "/usr/bin/serf"
 
 	// JournalctlBin is the default location of the journalctl inside planet
-	JournalctlBin = "/usr/bin/journalctl"
+	JournalctlBin = "/bin/journalctl"
```

Comment:
This path changed in planet at some point. I did not find a ticket for this but will probably need to create and verify that it's been consistently fixed in active branches.